common/api/components-react.api.md

```diff
@@ -3249,6 +3367,32 @@ export function useVirtualizedPropertyGridLayoutStorage<T extends Element>(): {
     restore: () => void;
 };
 
+// @beta
+export type Value = NumericValue | InstanceKeyValue | TextValue | BooleanValue | DateValue | EnumValue;
+
+// @beta (undocumented)
+export namespace Value {
```

Comment:
Should we avoid adding new namespaces? https://typescript-eslint.io/rules/no-namespace/
Add a new module `ValueUtilities.ts` instead and export it in the barrel file `export * as ValueUtilities from "ValueUtilities.js"`

Reply:
Using the module aggregation method described above will reduce the clutter without using the namespace:

```tsx
// ValueUtilities.ts
export const x = 0;
export const y = 0;

// barrel.ts
export * as ValueUtilities from "ValueUtilities.js";

// Consumer usage
import { ValueUtilities } from "components-react";
ValueUtilities.x;
```